### PR TITLE
nav bar with two themes

### PR DESCRIPTION
**Pull request by quiz1991**
screen shot 2017-03-29 at 11 12 40 pm
screen shot 2017-03-29 at 11 12 50 pm

@anthonyseu @jacgrady1 
Still working on transition while scrolling.

### CODE REVIEW

File: components/Nav.js [hunk 1]
```diff
@@ -7,44 +7,139 @@ import UserActions from '../actions/UserActions';
...
         this.context.executeAction(UserActions.navToLogin, {});
     }
     navToSignup() {
         this.context.executeAction(UserActions.navToSignup, {});
     }
+    navToHome() {
+        this.context.executeAction(UserActions.navToHome, {});
+    }
+    componentDidMount() {
+        // window.addEventListener('scroll', this.handleScroll.bind(this));
```
Comment (jacgrady1) [inline components/Nav.js]: still working on this?

Comment (quiz1991) [inline components/Nav.js]: yep.

File: components/partials/SearchButton.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+    handleChange(e) {
+        if (this.props.onChange) {
+            this.props.onChange(e)
+        }
+    }
+    render() {
+        return (
+            <RaisedButton
+            //   label="Search"
```
Comment (jacgrady1) [inline components/partials/SearchButton.js]: remove?

Comment (quiz1991) [inline components/partials/SearchButton.js]: ohh sure.

File: components/Nav.js [hunk 3]
```diff
@@ -7,44 +7,139 @@ import UserActions from '../actions/UserActions';
 import UserActionMenu from './UserActionMenu';
 import FlatButton from 'material-ui/FlatButton';
 import SearchBar from './partials/SearchBar';
+import {tftYellow, tftBlue, tftWhite} from '../config/colorConstants';
 
-const buttonStyle = {
-    height: '50px'
-};
+function getStyles (isExtended) {
```
Comment (jacgrady1) [inline components/Nav.js]: 🥇 good practice!

**Comment by quiz1991** [comment]
@jacgrady1 done with review?

**Comment by jacgrady1** [comment]
@ I think you are still working on this right?

**Comment by quiz1991** [comment]
@jacgrady1 Oh sorry. i mean we can merge it for now and i'll make another pr for the transition.

**Pull request merged**